Automattic/wp-calypso
# Fix LiB and newsletter intro design

## Proposed Changes

Small changes were correctly pointed on this issue #66593 
This PR should fix these issues for Link in bio and Newsletter flows.

## Testing Instructions
- Pull this branch and run `yarn start` from the root folder
- Navigate to http://calypso.localhost:3000/setup/intro?flow=link-in-bio to check Link in bio design
- Navigate to http://calypso.localhost:3000/setup/intro?flow=newsletter to check Newsletter design
- All the items addressed on here #66593 should be fixed.

Related to #66593
Fixes #66593
